src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+
+  let price = days * dayPrice;
```

Comment:
Consider using `const` instead of `let` for the `price` variable since it's not being reassigned but only modified within the `if` blocks. This helps to prevent unintentional reassignment and makes the code easier to reason about.